src/services/API/sqs.js

```diff
@@ -0,0 +1,50 @@
+import { SQSClient, SendMessageCommand } from "@aws-sdk/client-sqs";
+
+const REGION = "ap-south-1";
+const queueURL = "https://sqs.ap-south-1.amazonaws.com/111766607077/EventQueue";
```

Comment:
Use all caps just like `REGION`. This is a good practice to distinguish the global variables from local. But it's more impirtant to be consistent, so you can rename `REGION` to `region` also :)